content/ADR-125.md

```diff
@@ -0,0 +1,208 @@
+---
+adr: 125
+title: User interface components for SDK 7
+date: 2022-11-14
+status: Living
```

Comment:
Given the status is already `Living`, perhaps some of the wording of the whole doc could be changed from future tense to present / past. e.g. `the name will be changed to PointerEvents` -> `the name has changed to PointerEvents`.